docs/source/daml/intro/7_Composing.rst

```diff
@@ -211,18 +211,28 @@ DAML's execution model
 
 DAML's execution model is fairly easy to understand, but has some important consequences. You can imagine the life of a transaction as follows:
 
-1. A party submits a list of ``Commands``.
-2. The transaction is interpreted, meaning the ``Update`` corresponding to each command is evaluated in the context of the ledger to calculate all consequences, including transitive ones (consequences of consequences, etc.).
-3. The views of the transaction that parties get to see (see :ref:`privacy`) are calculated in a process called *blinding*, or *projecting*.
-4. The blinded views are distributed to the parties.
-5. The transaction is *validated* based on the blinded views and a consensus protocol depending on the underlying infrastructure.
-6. If validation succeeds, the transaction is *committed*.
+Command
+  A user submits a list of Commands via the Ledger API of a Participant Node, acting as a `Party` hosted on that Node. That party is called the requester.
+Interpretation
+  Each Command corresponds to one or more Actions. During this step, the ``Update`` corresponding to each Action is evaluated in the context of the ledger to calculate all consequences, including transitive ones (consequences of consequences, etc.). The result of this is a complete Transaction. Together with its requestor, this is also known as a Commit.
+Blinding
+  On ledgers with strong privacy, projections (see :ref:`privacy`) for all involved parties are created. This is also called *projecting*.
+Submission
+  The Transaction/Commit is submitted to the network.
```

Comment:
This sounds like you call submission only the part where the participant sends it? This is slightly confusing since you call a Submit endpoint when you are sending it _to_ the participant as well.

Reply:
It's true, the service is called `CommandSubmission`, and the requests things like `SubmitAndWait`. We are overall not that consistent in our terminology. Maybe I'll call this transaction submission just to be super clear.